tests/data/decorators.py

```diff
@@ -0,0 +1,176 @@
+# This file doesn't use the standard decomposition
+# decorator syntax test cases are separated by double # comments
+# Those before the 'output' comment are valid under the old syntax
+# Those after the 'ouput' comment require PEP614 relaxed syntax
+# Do not remove the double # separator before the first test case, it allows
+# to the comment before the test case
```

Comment:
remove "to"?

Reply:
I think I also forgot the verb. It should be fixed now. Thanks !